app/src/main/java/net/aiscope/gdd_app/ui/sample_completion/metadata/MetadataFragment.kt

```diff
@@ -46,21 +46,29 @@ class MetadataFragment : SampleFormFragment, Fragment(R.layout.fragment_metadata
 
                 smearTypeId?.let { metadataSectionSmearTypeRadioGroup.check(it) }
                 speciesValue?.let { metadataSpeciesSpinner.select(it) }
+                comments?.let {
+                    metadataCommentsInput.setText(it)
+                }
             }
         }
     }
 
-    override fun validateAndUpdateVM(): Boolean {
+    fun updateVM() {
         with(binding) {
             sharedVM.smearTypeId = metadataSectionSmearTypeRadioGroup.checkedRadioButtonId
             sharedVM.speciesValue = metadataSpeciesSpinner.selectedItem.toString()
             sharedVM.comments = metadataCommentsInput.text.toString()
         }
+    }
+    override fun validateAndUpdateVM(): Boolean {
+        updateVM()
         //No validation on this tab
         return true
     }
 
     private fun onAddImageClicked() {
+        updateVM()

```

Comment:
This is the main change introduced. Saving without validating. 

Now to find an easy way for the model to handle its sanity, before sending to remote.